lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -472,6 +472,39 @@ async function managerUpdateArtifacts(
   manager: string,
   updateArtifact: UpdateArtifact,
 ): Promise<UpdateArtifactsResult[] | null> {
+  const updatedDepsCount = updateArtifact.updatedDeps.length;
+  if (updatedDepsCount === 0) {
+    // When there are no updated dependencies, use the config's value.
+    if (updateArtifact.config.skipArtifactUpdating) {
+      logger.debug(
+        { manager, packageFileName: updateArtifact.packageFileName },
+        'skipping artifact updating because skipArtifactUpdating=true',
+      );
+      return null;
+    }
+  } else {
+    // When there are updated dependencies, use the dependency's value.
+    const skipArtifactUpdatingCount = updateArtifact.updatedDeps.filter(
```

Comment:
ℹ️ Instead of using `updateArtifact.updatedDeps.every(...)` to determine if all of the updated deps want to skip artifact updating, I decided to get the number of updates deps with `skipArtifactUpdating` set to true so that a log message could indicate why artifact updating was going to run. For example, instead of just _not_ seeing the "skipping artifact updating" message, you will see a message like:

> running artifact updating because 2 out of 3 updated dependencies have skipArtifactUpdating=false

I thought this would be better for debugging because if you're expecting artifact updating to be skipped, but it's still running, then seeing _why_ it's running will make it easier to work out what's going on.